libs/portals/admin/ids-admin/src/components/forms/EditApplication/EditApplication.action.ts

```diff
@@ -34,6 +35,23 @@ const checkIfStringIsListOfUrls = (urls: string) => {
   return true
 }
 
+const checkIfStringIsArrayOfValidClaims = (claims: string) => {
+  if (claims.length === 0) {
```

Comment:
I would check it as, as it checks for undefined, null and empty string.
```suggestion
  if (!claims) {
```